pom.xml

```diff
@@ -6,8 +6,7 @@
     <parent>
         <groupId>dev.ikm.tinkar</groupId>
         <artifactId>tinkar-data-bom</artifactId>
-        <version>1.2.0</version>
-        <relativePath/>
+        <version>1.3.0-SNAPSHOT</version>
```

Comment:
After the new release of tinkar-data-bom that contains the updated tinkar-core version, we should change this to 1.3.0.